src/Algebra/Consequences/Base.agda

```diff
@@ -7,27 +7,40 @@
 
 {-# OPTIONS --cubical-compatible --safe #-}
 
+open import Relation.Binary.Core using (Rel)
+
 module Algebra.Consequences.Base
-  {a} {A : Set a} where
+  {a ℓ} {A : Set a} where
 
-open import Algebra.Core
+open import Algebra.Core using (Op₁; Op₂)
 open import Algebra.Definitions
-open import Data.Sum.Base
-open import Relation.Binary.Core
+open import Data.Sum.Base using (reduce)
 open import Relation.Binary.Definitions using (Reflexive)
 
-module _ {ℓ} {_•_ : Op₂ A} (_≈_ : Rel A ℓ) where
+private
+  variable
+    f : Op₁ A
+    _∙_ : Op₂ A
+
+
+------------------------------------------------------------------------
+-- Selective
+
+module _  (_≈_ : Rel A ℓ) where
 
-  sel⇒idem : Selective _≈_ _•_ → Idempotent _≈_ _•_
+  sel⇒idem : Selective _≈_ _∙_ → Idempotent _≈_ _∙_
   sel⇒idem sel x = reduce (sel x x)
 
-module _ {ℓ} {f : Op₁ A} (_≈_ : Rel A ℓ) where
+------------------------------------------------------------------------
+-- SelfInverse
+
+module _  (_≈_ : Rel A ℓ) where
```

Comment:
Does there need to be two separate modules here? Can/should the `module _ (_\~~_ : Rel A \ell) where` headers be merged?

Reply:
Hmmm... yes, but I had been trying to restore `Algebra.Consequences.base` to as-near-as-possible its original form, which had the two distinct internal modules.

Waiting on @MatthewDaggitt 's review comments on the two PRs split out of the orginal... but could indeed make your suggested change!

Reply:
Resolving in favour of addressing the equality refactoring in #2572